assets/js/components/ExecutionResults/CheckResultDetail/BackToTargetExecution.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { TARGET_CLUSTER, TARGET_HOST } from '@lib/model';
+
+import BackButton from '@components/BackButton';
+
+function BackToTargetExecution({ targetID, targetType }) {
+  let targetExecutionURL;
```

Comment:
Thoughts on using a map to do this, rather than a `switch` statement?

e.g.

```javascript
const targetTypeToExecutionURL = {
  [TARGET_CLUSTER]: `/clusters/${targetID}/executions/last`,
  [TARGET_HOST]: `/hosts/${targetID}/executions/last`
}

const targetExecutionURL = targetTypeToExecutionURL[targetType] ?? null
```

Variables names up for debate :stuck_out_tongue_winking_eye:

Reply:
Alright. I actually did that in some other place like https://github.com/trento-project/web/blob/main/assets/js/components/ExecutionResults/CheckResultDetail/CheckResultInfoBox.jsx :see_no_evil: